lib/MusicBrainz/Server/Controller/Account.pm

```diff
@@ -145,7 +146,13 @@ sub lost_password : Path('/lost-password') ForbiddenOnSlaves
     my ($self, $c) = @_;
 
     if (exists $c->request->params->{sent}) {
-        $c->stash(template => 'account/lost_password_sent.tt');
+        $c->stash(
+            current_view => 'Node',
+            component_path => 'account/LostPasswordSent',
+            component_props => {
+                contactURL => $CONTACT_URL,
```

Comment:
Should we define it in root/static/scripts/common/constants.js instead of passing it as a prop?

Reply:
So be it with https://github.com/metabrainz/musicbrainz-server/pull/699/commits/8b5059909982e2bbf8bde1493b2c2547495c6a29.
That’s definitely better than passing it as a prop.
Should we define it outside of `root/static` as it is not useful to client-side scripts?

Reply:
Hmm, sure, it sounds useful to have a root/constants that's not bundled for the client. (Haven't checked if anything else could be moved there.)

Reply:
Done in https://github.com/metabrainz/musicbrainz-server/pull/699/commits/5b775265bd5497e9ed5bf3cec03dcf69d7ac5844 and https://github.com/metabrainz/musicbrainz-server/pull/699/commits/e0f9d92c4f5e34bfde9fe66f9564c42e73d46e39.